Styling/schema/style.meta.schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema" : "http://json-schema.org/draft-03/schema",
+    "title" : "meta",
+    "type" : "object",
+    "description" : "A series of property names and the expression to evaluate for the value of each property.",
+    "properties" : {
+    },
+    "additionalProperties" : {
+        "$ref" : "style.metaProperty.schema.json"
```

Comment:
OK, perhaps that is sufficient for this schema doc, but I still think we need to think through the interop between the styling language types (returned by the expression) and JavaScript/Cesium types.

What do you think about adding a small section to the spec about this?  Basically:
* `Boolean`, `Null`, `Undefined`, `Number`, and `String` map directly to JavaScript types.
* `Color` maps to `Cesium.Color`
* `RegExp` maps to what?  Converted to a string with its `toString` function?  Converted to a JavaScript `RegExp`?  Is in valid?